mono/eglib/eglib-remap.h

```diff
@@ -311,3 +311,6 @@
 #define g_set_printerr_handler monoeg_set_printerr_handler
 
 #define g_size_to_int monoeg_size_to_int
+#define g_ascii_charcmp monoeg_ascii_charcmp
```

Comment:
Aren't these static? Why is this remapping happening? `g_warning_d` doesn't even matter outside of wasm, where it exists to limit stack size.

Reply:
This is there to fix test-eglib-remap.

Reply:
Not sure why a static function shows up here.

Reply:
Would be nice to know what's going on there, that seems strange.